app/services/create_payment.rb

```diff
@@ -0,0 +1,9 @@
+class CreatePayment
+
+  def call(event)
+    charge = event.data.object
+    user = User.find_by(stripe_customer_id: charge.customer)
+    Payment.create(amount_cents: charge.amount, user: user, reference_id: charge.id) if user
```

Comment:
If you are not wrapping the create with an `if`/`else` there is a decent chance you might want to know if it fails. To raise an exception, use `create!` instead.